### PR TITLE
New mapviews graph for map cards

### LINKED ISSUE #3135: Rethink how to show map activity

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2015-04-27 at 17 31 25

- [x] Bars graph instead of a line graph.
- [x] Tooltip on hover (only for bars with value).

Fixes #3135

cc @saleiva

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3306/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3307/))

**Comment by xavijam** [comment]
@javierarce could you review it please :) ?

File: lib/assets/javascripts/cartodb/new_common/dialogs/create/create_content.js [hunk 1]
```diff
@@ -106,7 +106,6 @@ module.exports = cdb.core.View.extend({
     if (this._onboardingView) {
       this.addView(this._onboardingView);
     }
-    this._maybeShowOnboarding();
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_common/dialogs/create/create_content.js]: Not related with this pull request, but fixing a double onboarding dialog in new-create-dialog :S.

File: lib/assets/javascripts/cartodb/new_dashboard/onboarding_view.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ module.exports = cdb.core.View.extend({
   },
 
   render: function() {
-    this.$el.append(
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/onboarding_view.js]: Better always html than append, preventing double elements (not related with this PR again, sorry).

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3308/))

File: lib/assets/javascripts/cartodb/new_dashboard/mapviews_graph.js [hunk 3]
```diff
@@ -6,67 +11,95 @@
...
+        .attr("y", function(d) {
+          var value = height - y(d.mapviews);
+          var yPos = y(d.mapviews);
+          return value < minHeight ? (height - minHeight) : yPos;
+        })
+        .attr("height", function(d) {
+          var value = height - y(d.mapviews);
+          return value < minHeight ? minHeight : value;
+        })
+        .on('mouseover', function(d) {
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/mapviews_graph.js]: I would move add a method for these mouse events:

```.on('mouseover', onMouseOver)```
```.on('mouseout', onMouseOut)```

Just a suggestion and not very important (so, feel free to ignore it)

**Comment by javierarce** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3311/))

**Pull request merged**